# Chinese (Taiwan) translation for apport
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-08-19 15:19+0200\n"
"PO-Revision-Date: 2015-04-27 21:47+0000\n"
"Last-Translator: Martin Pitt <martin@piware.de>\n"
"Language-Team: Chinese (Taiwan) <zh_TW@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2022-08-19 12:20+0000\n"
"X-Generator: Launchpad (build 87919ae1fbf1142b0e02b80650e6ccf3e775beb3)\n"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr "蒐集系統資訊"

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr "若要為此問題回報蒐集系統資訊需先通過身份驗證"

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr ""

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid "Please enter your password to access problem reports of system programs"
msgstr ""

#: ../apport/ui.py:164
msgid "This package does not seem to be installed correctly"
msgstr "這個軟體包似乎尚未正確安裝"

#: ../apport/ui.py:174
#, python-format
msgid ""
"This does not seem to be an official %s package. Please retry after updating "
"the indexes of available packages, if that does not work then remove related "
"third party packages and try again."
msgstr ""

#: ../apport/ui.py:204
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"您安裝了某些已過時的軟體包。請升級以下軟體包以及檢查問題是否繼續產生：\n"
"\n"
"%s"

#: ../apport/ui.py:358
msgid "unknown program"
msgstr "未知程式"

#: ../apport/ui.py:361
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "對不起，\"%s\" 程式已不正常關閉"

#: ../apport/ui.py:364 ../apport/ui.py:1898
#, python-format
msgid "Problem in %s"
msgstr "在 %s 中的問題"

#: ../apport/ui.py:369
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr "您的電腦並沒有足夠的可用記憶體可自動分析問題並傳送給開發者一份報告。"

#: ../apport/ui.py:429 ../apport/ui.py:447 ../apport/ui.py:620
#: ../apport/ui.py:629 ../apport/ui.py:866 ../apport/ui.py:1676
#: ../apport/ui.py:1859 ../apport/ui.py:1864
msgid "Invalid problem report"
msgstr "無效的問題回報"

#: ../apport/ui.py:430
msgid "You are not allowed to access this problem report."
msgstr "您並不被允許存取此份問題回報。"

#: ../apport/ui.py:438
msgid "Error"
msgstr "錯誤"

#: ../apport/ui.py:440
msgid "There is not enough disk space available to process this report."
msgstr "沒有足夠的磁碟空間以處理此份報告。"

#: ../apport/ui.py:478
msgid "No PID specified"
msgstr "沒有指定 PID"

#: ../apport/ui.py:480
msgid "You need to specify a PID. See --help for more information."
msgstr "您需要指定一個 PID。更多說明請見 --help。"

#: ../apport/ui.py:491 ../apport/ui.py:596
msgid "Invalid PID"
msgstr "無效的 PID"

#: ../apport/ui.py:492
msgid "The specified process ID does not exist."
msgstr "指定的程序 ID 不存在。"

#: ../apport/ui.py:497
msgid "Not your PID"
msgstr "不是您的 PID"

#: ../apport/ui.py:498
msgid "The specified process ID does not belong to you."
msgstr "指定的程序 ID 不屬於您。"

#: ../apport/ui.py:553
msgid "No package specified"
msgstr "未指定軟體包"

#: ../apport/ui.py:555
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr "您需要指定一個套件或一個 PID。更多資訊請參考 --help。"

#: ../apport/ui.py:583
msgid "Permission denied"
msgstr "權限不足"

#: ../apport/ui.py:585
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr "指定的程序並不屬於您。請以程序擁有者或是 root 身份執行此程式。"

#: ../apport/ui.py:598
msgid "The specified process ID does not belong to a program."
msgstr "指定的程序 ID 並不屬於一個程式。"

#: ../apport/ui.py:622
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr "徵狀判斷指令稿 %s 無法判斷出受影響的軟體包"

#: ../apport/ui.py:630
#, python-format
msgid "Package %s does not exist"
msgstr "%s 軟體包不存在"

#: ../apport/ui.py:659 ../apport/ui.py:871 ../apport/ui.py:904
#: ../apport/ui.py:914
msgid "Cannot create report"
msgstr ""

#: ../apport/ui.py:674 ../apport/ui.py:733 ../apport/ui.py:751
msgid "Updating problem report"
msgstr ""

#: ../apport/ui.py:676
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""

#: ../apport/ui.py:688
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""

#: ../apport/ui.py:734 ../apport/ui.py:752
msgid "No additional information collected."
msgstr ""

#: ../apport/ui.py:815
msgid "What kind of problem do you want to report?"
msgstr "您想要回報何種問題？"

#: ../apport/ui.py:836
msgid "Unknown symptom"
msgstr "未知的症狀"

#: ../apport/ui.py:837
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "症狀 %s 未知。"

#: ../apport/ui.py:873
msgid ""
"The window option cannot be used on Wayland.\n"
"\n"
"Please find the window's process ID and then run 'ubuntu-bug <process ID>'.\n"
"\n"
"The process ID can be found by running the System Monitor application. In "
"the Processes tab, scroll until you find the correct application. The "
"process ID is the number listed in the ID column."
msgstr ""

#: ../apport/ui.py:889
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""

#: ../apport/ui.py:906 ../apport/ui.py:915
msgid "xprop failed to determine process ID of the window"
msgstr ""

#: ../apport/ui.py:930
#, python-format
msgid "%(prog)s <report number>"
msgstr ""

#: ../apport/ui.py:931
msgid "Specify package name."
msgstr "指定軟體包名稱。"

#: ../apport/ui.py:937 ../apport/ui.py:1069
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr ""

#: ../apport/ui.py:975
#, fuzzy, python-format
msgid ""
"%(prog)s [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr "%prog [options] [symptom|pid|package|program path|.apport/.crash file]"

#: ../apport/ui.py:986
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""
"啟動為錯誤歸檔模式。需要 --packages 和一個可選擇的 --pid，或是只需 --pid。若"
"兩者都未給予，顯示一份已知症狀的列表。（暗示若只有給予單一參數。）"

#: ../apport/ui.py:995
msgid "Click a window as a target for filing a problem report."
msgstr ""

#: ../apport/ui.py:1004
msgid "Start in bug updating mode. Can take an optional --package."
msgstr ""

#: ../apport/ui.py:1013
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr "建立關於症狀的錯誤報告。(暗示是否只要提供症狀名稱作為引數。)"

#: ../apport/ui.py:1022
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"在 --file-bug 模式中指定套件名稱。若指定 --pid 則是可選擇的。（暗示若套件名稱"
"是唯一給予的參數。）"

#: ../apport/ui.py:1033
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""

#: ../apport/ui.py:1041
msgid "The provided pid is a hanging application."
msgstr "提供的 pid 是懸凍的應用程式。"

#: ../apport/ui.py:1049
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""
"從特定的 .apport 或 .crash 檔案來回報當掉情形，代替等待中的 %s。（暗示若特定"
"檔案是唯一給予的參數。）"

#: ../apport/ui.py:1059
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""

#: ../apport/ui.py:1077
msgid "Print the Apport version number."
msgstr "列印 Apport 版本號碼。"

#: ../apport/ui.py:1239
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr "這會在終端機視窗中啟動 apport-retrace 以檢查程式當掉原因。"

#: ../apport/ui.py:1243
msgid "Run gdb session"
msgstr "執行 gdb 作業階段"

#: ../apport/ui.py:1244
msgid "Run gdb session without downloading debug symbols"
msgstr "執行 gdb 作業階段而不下載除錯符號"

#. TRANSLATORS: %s contains the crash report file name
#: ../apport/ui.py:1246
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr "以完整的符號堆疊追蹤來更新 %s"

#: ../apport/ui.py:1308
msgid "Can't remember send report status settings"
msgstr ""

#: ../apport/ui.py:1312
msgid ""
"Saving crash reporting state failed. Can't set auto or never reporting mode."
msgstr ""

#: ../apport/ui.py:1395 ../apport/ui.py:1408
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr "此問題報告用於不再安裝的程式。"

#: ../apport/ui.py:1432
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr "發生問題的程式 %s 自上次遭遇當掉之後已有變動。"

#: ../apport/ui.py:1493 ../apport/ui.py:1617 ../apport/ui.py:1868
msgid "This problem report is damaged and cannot be processed."
msgstr "此問題回報已損壞且無法處理。"

#: ../apport/ui.py:1502
msgid "This report is about a package that is not installed."
msgstr ""

#: ../apport/ui.py:1510
msgid "An error occurred while attempting to process this problem report:"
msgstr "試圖處理問題回報時遭遇錯誤："

#: ../apport/ui.py:1527
msgid ""
"You have two versions of this application installed, which one do you want "
"to report a bug against?"
msgstr "此應用程式您安裝了兩個版本，您想對哪一個版本回報臭蟲？"

#: ../apport/ui.py:1532
#, python-format
msgid "%s snap"
msgstr "%s snap"

#: ../apport/ui.py:1533
#, python-format
msgid "%s deb package"
msgstr "%s deb 套件包"

#: ../apport/ui.py:1574
#, python-format
msgid "%s is provided by a snap published by %s. Contact them via %s for help."
msgstr ""

#: ../apport/ui.py:1579
#, python-format
msgid ""
"%s is provided by a snap published by %s. No contact address has been "
"provided; visit the forum at https://forum.snapcraft.io/ for help."
msgstr ""
"%s 是由 %s 所發布的 snap 。開發者沒有提供聯絡地址；請前往論壇 https://forum."
"snapcraft.io/ 尋求協助。"

#: ../apport/ui.py:1678
msgid "Could not determine the package or source package name."
msgstr "無法決定軟體包或原始碼軟體包名稱。"

#: ../apport/ui.py:1703
msgid "Unable to start web browser"
msgstr "無法啟動網頁瀏覽器"

#: ../apport/ui.py:1704
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "無法啟動網頁瀏覽器以開啟 %s。"

#: ../apport/ui.py:1814
msgid "Network problem"
msgstr "網路問題"

#: ../apport/ui.py:1818
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr "無法連接到當機資料庫，請檢查您的網路連線。"

#: ../apport/ui.py:1850
msgid "Memory exhaustion"
msgstr "記憶體耗盡"

#: ../apport/ui.py:1852
msgid "Your system does not have enough memory to process this crash report."
msgstr "您的系統沒有足夠的記憶體來處理此份當機報告。"

#: ../apport/ui.py:1903
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"此問題無法被回報：\n"
"\n"
"%s"

#: ../apport/ui.py:1959 ../apport/ui.py:1971
msgid "Problem already known"
msgstr "已知問題"

#: ../apport/ui.py:1961
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"此問題在網路瀏覽器顯示之錯誤報告中已回報。請檢查您是否能新增任何可能對開發者"
"有幫助的進一步資訊。"

#: ../apport/ui.py:1973
msgid "This problem was already reported to developers. Thank you!"
msgstr "此問題已經回報給開發者。感謝您！"

#: ../bin/apport-cli.py:84
msgid "Press any key to continue..."
msgstr "按任意鍵繼續..."

#: ../bin/apport-cli.py:91
msgid "What would you like to do? Your options are:"
msgstr "您要做什麼？選擇有："

#: ../bin/apport-cli.py:104
#, python-format
msgid "Please choose (%s):"
msgstr "請選擇 (%s):"

#: ../bin/apport-cli.py:172
#, python-format
msgid "(%i bytes)"
msgstr ""

#: ../bin/apport-cli.py:174 ../gtk/apport-gtk.py:168 ../kde/apport-kde.py:401
msgid "(binary data)"
msgstr "（二進制資料）"

#: ../bin/apport-cli.py:208 ../gtk/apport-gtk.py:207 ../gtk/apport-gtk.ui.h:6
#: ../kde/apport-kde.py:195
msgid "Send problem report to the developers?"
msgstr "要將傳送問題報告給開發者嗎？"

#: ../bin/apport-cli.py:210
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"在問題報告傳送後，請填寫自動開啟之\n"
"網路瀏覽器中的表格。"

#: ../bin/apport-cli.py:217
#, python-format
msgid "&Send report (%s)"
msgstr "傳送報告 (%s)(&S)"

#: ../bin/apport-cli.py:222
msgid "&Examine locally"
msgstr "於本地端檢查(&E)"

#: ../bin/apport-cli.py:226
msgid "&View report"
msgstr "檢視報告(&V)"

#: ../bin/apport-cli.py:229
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr "保留回報檔，稍後傳送或複製至其他位置(&K)"

#: ../bin/apport-cli.py:234
msgid "Cancel and &ignore future crashes of this program version"
msgstr "取消及忽略未來此程式版本的當機(&I)"

#: ../bin/apport-cli.py:237 ../bin/apport-cli.py:331 ../bin/apport-cli.py:365
#: ../bin/apport-cli.py:388
msgid "&Cancel"
msgstr "取消(&C)"

#: ../bin/apport-cli.py:272
msgid "Problem report file:"
msgstr "問題回報檔："

#: ../bin/apport-cli.py:278 ../bin/apport-cli.py:283
msgid "&Confirm"
msgstr "確認(&C)"

#: ../bin/apport-cli.py:282
#, python-format
msgid "Error: %s"
msgstr "錯誤：%s"

#: ../bin/apport-cli.py:288 ../kde/apport-kde.py:456
msgid "Collecting problem information"
msgstr "正在收集問題資訊"

#: ../bin/apport-cli.py:290
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"收集的資訊可傳送予開發者以改進此\n"
"應用程式。這可能需要數分鐘。"

#: ../bin/apport-cli.py:306 ../gtk/apport-gtk.ui.h:17 ../kde/apport-kde.py:504
msgid "Uploading problem information"
msgstr "正在上傳問題資訊"

#: ../bin/apport-cli.py:308
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"已收集的資訊將傳送至臭蟲追蹤系統。\n"
"這可能需要數分鐘。"

#: ../bin/apport-cli.py:364
msgid "&Done"
msgstr "完成(&D)"

#: ../bin/apport-cli.py:370
msgid "none"
msgstr "無"

#: ../bin/apport-cli.py:372
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "已選擇：%s。多種選擇："

#: ../bin/apport-cli.py:389
msgid "Choices:"
msgstr "選項："

#: ../bin/apport-cli.py:403
msgid "Path to file (Enter to cancel):"
msgstr "檔案路徑 (按 Enter 取消)："

#: ../bin/apport-cli.py:409
msgid "File does not exist."
msgstr "檔案不存在。"

#: ../bin/apport-cli.py:411
msgid "This is a directory."
msgstr "這是個目錄。"

#: ../bin/apport-cli.py:417
msgid "To continue, you must visit the following URL:"
msgstr "欲繼續，您必須造訪以下網址："

#: ../bin/apport-cli.py:420
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr "您現在可以啟動瀏覽器，或複製此網址到另一臺電腦的瀏覽器中。"

#: ../bin/apport-cli.py:426
msgid "Launch a browser now"
msgstr "現在啟動瀏覽器"

#: ../bin/apport-cli.py:443
msgid "No pending crash reports. Try --help for more information."
msgstr "無等待中的當機報告。更多資訊請試試 --help。"

#: ../bin/apport-retrace.py:44
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr "不要將新的錯誤追溯放在報告中，但輸出至標準輸出（stdout）。"

#: ../bin/apport-retrace.py:53
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr "啟動新的互動式 gdb 連線階段及核心傾印報告（-o 忽略：不會複寫報告）"

#: ../bin/apport-retrace.py:62
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr "寫入已修改的報告至特定的檔案來替代改變原始報告"

#: ../bin/apport-retrace.py:72
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr "在堆疊追溯重新產生後從報告中移除核心傾印"

#: ../bin/apport-retrace.py:80
msgid "Override report's CoreFile"
msgstr "覆蓋報告的核心檔案（CoreFile）"

#: ../bin/apport-retrace.py:86
msgid "Override report's ExecutablePath"
msgstr "覆蓋報告的可執行路徑（ExecutablePath）"

#: ../bin/apport-retrace.py:92
msgid "Override report's ProcMaps"
msgstr "覆蓋報告的行程對映（ProcMaps）"

#: ../bin/apport-retrace.py:98
msgid "Rebuild report's Package information"
msgstr "重建報告的軟體包資訊"

#: ../bin/apport-retrace.py:105
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""
"建立暫時沙箱，並下載/安裝需要的軟體包與除錯符於該沙箱中；若不採用此選項，將假"
"定需要的軟體包與除錯符都已經安裝在系統之內。引數指向打包系統組態的基礎目錄；"
"如果您有指定「system」(系統) 的話，它會使用系統組態檔，但是接著只能回追發生於"
"目前執行中發行版的當掉情形。"

#: ../bin/apport-retrace.py:119
msgid ""
"Build another temporary sandbox for installing gdb and its dependencies "
"using the same release as the report rather than whatever version of gdb you "
"have installed."
msgstr ""
"建置另一個暫時沙盒以安裝 gdb 和其相關依賴以使用相同發行版作回報，而不是採用您"
"本機已安裝的 gdb 版本。"

#: ../bin/apport-retrace.py:129
msgid "Report download/install progress when installing packages into sandbox"
msgstr "當軟體包安裝至沙箱時回報下載/安裝進度"

#: ../bin/apport-retrace.py:136
msgid "Prepend timestamps to log messages, for batch operation"
msgstr "在記錄訊息前加上時間戳記，供批次操作"

#: ../bin/apport-retrace.py:142
msgid ""
"Create and use third-party repositories from origins specified in reports"
msgstr "製作並使用指定的第三方軟體庫來源置於報告中"

#: ../bin/apport-retrace.py:150
msgid "Cache directory for packages downloaded in the sandbox"
msgstr "為下載於沙箱內的軟體包製作目錄快取"

#: ../bin/apport-retrace.py:156
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""
"解開軟體包的目錄。未來執行時將假定任何已下載的軟體包也都抽出於此沙盒中。"

#: ../bin/apport-retrace.py:167 ../bin/apport-valgrind.py:97
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr "安裝額外軟體包至沙箱內 (可以指定多次)"

#: ../bin/apport-retrace.py:174
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""

#: ../bin/apport-retrace.py:184
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""

#: ../bin/apport-retrace.py:192
msgid "Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr "重複的 sqlite 資料庫路徑（預設：無重複檢查）"

#: ../bin/apport-retrace.py:200
msgid "Do not add StacktraceSource to the report."
msgstr ""

#: ../bin/apport-retrace.py:213
msgid "You cannot use -C without -S. Stopping."
msgstr ""

#. translators: don't translate y/n,
#. apport currently only checks for "y"
#: ../bin/apport-retrace.py:243
msgid "OK to send these as attachments? [y/n]"
msgstr "允許傳送這些附件？"

#: ../bin/apport-unpack.py:29
#, fuzzy, python-format
msgid "%(prog)s <report> <target directory>"
msgstr "用法：%s <回報檔> <目標目錄>"

#: ../bin/apport-unpack.py:31
msgid "Report file to unpack"
msgstr ""

#: ../bin/apport-unpack.py:33
msgid "directory to unpack report to"
msgstr ""

#: ../bin/apport-unpack.py:45
msgid "Destination directory exists and is not empty."
msgstr "目的目錄存在且不是空的。"

#: ../bin/apport-valgrind.py:37
msgid "See man page for details."
msgstr ""

#: ../bin/apport-valgrind.py:46
msgid "specify the log file name produced by valgrind"
msgstr ""

#: ../bin/apport-valgrind.py:52
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""

#: ../bin/apport-valgrind.py:60
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""
"不能新增或使用之前的沙盒給額外的Debug symbol 用途,但僅依靠安裝的 debug "
"symbols"

#: ../bin/apport-valgrind.py:69
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""

#: ../bin/apport-valgrind.py:78
msgid "report download/install progress when installing packages into sandbox"
msgstr "當軟體包安裝至沙箱時回報下載/安裝進度"

#: ../bin/apport-valgrind.py:86
msgid ""
"the executable that is run under valgrind's memcheck tool for memory leak "
"detection"
msgstr ""

#: ../bin/apport-valgrind.py:129
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr ""

#: ../data/apportcheckresume.py:68
msgid ""
"This occurred during a previous suspend, and prevented the system from "
"resuming properly."
msgstr "在上次暫停時遭遇這個問題，並使系統無法正常恢復。"

#: ../data/apportcheckresume.py:73
msgid ""
"This occurred during a previous hibernation, and prevented the system from "
"resuming properly."
msgstr "在上次休眠時遭遇這個問題，並使系統無法正常恢復。"

#: ../data/apportcheckresume.py:81
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr "回復行程在非常接近結束的時候停止回應，且將會表現出正常完成的情形。"

#: ../data/kernel_oops.py:30
msgid "Your system might become unstable now and might need to be restarted."
msgstr "您的系統現在也許會不穩定且需要重新啟動。"

#: ../gtk/apport-gtk.desktop.in.h:1 ../kde/apport-kde-mime.desktop.in.h:1
#: ../kde/apport-kde.desktop.in.h:1
msgid "Report a problem..."
msgstr "報告問題..."

#: ../gtk/apport-gtk.desktop.in.h:2 ../kde/apport-kde-mime.desktop.in.h:2
#: ../kde/apport-kde.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "報告問題予開發者"

#: ../gtk/apport-gtk.py:184
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr "很抱歉，應用程式 %s 已無預期停止。"

#: ../gtk/apport-gtk.py:187
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr "非常抱歉，%s 已無預警關閉。"

#: ../gtk/apport-gtk.py:192 ../kde/apport-kde.py:210 ../kde/apport-kde.py:262
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr "非常抱歉， %s 遭遇內部錯誤。"

#: ../gtk/apport-gtk.py:215 ../gtk/apport-gtk.ui.h:14 ../kde/apport-kde.py:203
msgid "Send"
msgstr "傳送"

#: ../gtk/apport-gtk.py:233 ../gtk/apport-gtk.py:665 ../gtk/apport-gtk.ui.h:11
#: ../kde/apport-kde.py:326
msgid "Show Details"
msgstr "顯示詳細資料"

#: ../gtk/apport-gtk.py:246 ../kde/apport-kde.py:254 ../kde/apport-kde.py:272
msgid "Continue"
msgstr "繼續"

#: ../gtk/apport-gtk.py:270
#, python-format
msgid "The application %s has stopped responding."
msgstr "應用程式 %s 已停止回應。"

#: ../gtk/apport-gtk.py:274
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr "程式「%s」已停止回應。"

#: ../gtk/apport-gtk.py:291 ../kde/apport-kde.py:218
#, python-format
msgid "Package: %s"
msgstr "軟體包：%s"

#: ../gtk/apport-gtk.py:297 ../kde/apport-kde.py:225
msgid "Sorry, a problem occurred while installing software."
msgstr "非常抱歉，於安裝軟體時遭遇問題。"

#: ../gtk/apport-gtk.py:309 ../gtk/apport-gtk.py:333 ../kde/apport-kde.py:234
#, python-format
msgid "The application %s has experienced an internal error."
msgstr "應用程式 %s 已遭遇某內部錯誤。"

#: ../gtk/apport-gtk.py:315 ../kde/apport-kde.py:241
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr "應用程式 %s 已無預警關閉。"

#: ../gtk/apport-gtk.py:346 ../gtk/apport-gtk.ui.h:7 ../kde/apport-kde.py:267
msgid "If you notice further problems, try restarting the computer."
msgstr "若您之後仍發現問題，請嘗試重新啟動電腦。"

#: ../gtk/apport-gtk.py:351 ../kde/apport-kde.py:274
msgid "Ignore future problems of this type"
msgstr "忽略未來發生的此類型問題"

#: ../gtk/apport-gtk.py:669 ../kde/apport-kde.py:323
msgid "Hide Details"
msgstr "隱藏詳細資料"

#: ../gtk/apport-gtk.ui.h:1 ../kde/apport-kde.py:545 ../kde/apport-kde.py:588
msgid "Apport"
msgstr ""

#: ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr ""

#: ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr "確定"

#: ../gtk/apport-gtk.ui.h:4
msgid "Crash report"
msgstr "當機回報"

#: ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr "<big><b>非常抱歉，發生內部錯誤。</b></big>"

#: ../gtk/apport-gtk.ui.h:8
msgid "Remember this in future"
msgstr ""

#: ../gtk/apport-gtk.ui.h:9
msgid "Ignore future problems of this program version"
msgstr "忽略本程式版本未來發生的錯誤"

#: ../gtk/apport-gtk.ui.h:10
msgid "Relaunch this application"
msgstr ""

#: ../gtk/apport-gtk.ui.h:12
msgid "_Examine locally"
msgstr "於本地端檢查(_E)"

#: ../gtk/apport-gtk.ui.h:13
msgid "Don't send"
msgstr ""

#: ../gtk/apport-gtk.ui.h:15
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>正在收集問題資訊</b></big>"

#: ../gtk/apport-gtk.ui.h:16
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr "正在收集資訊以幫助開發者修正您報告的問題。"

#: ../gtk/apport-gtk.ui.h:18
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>正在上傳問題資訊</b></big>"

#: ../gtk/apport-gtk.ui.h:19 ../kde/apport-kde.py:506
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr "正將收集到的資訊傳送至臭蟲追蹤系統。這可能需要數分鐘。"

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Apport 當機檔案"

#: ../kde/apport-kde.py:257
msgid "Leave Closed"
msgstr "維持關閉"

#: ../kde/apport-kde.py:258 ../kde/apport-kde.py:427
msgid "Relaunch"
msgstr "重新啟動"

#: ../kde/apport-kde.py:351
msgid "Username:"
msgstr ""

#: ../kde/apport-kde.py:352
msgid "Password:"
msgstr ""

#: ../kde/apport-kde.py:455
msgid "Collecting Problem Information"
msgstr "正收集問題資訊"

#: ../kde/apport-kde.py:458
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr "收集之資訊會傳送予開發者以改善該應用程式。此舉或需數分鐘。"

#: ../kde/apport-kde.py:503
msgid "Uploading Problem Information"
msgstr "正上傳問題資訊"
